survive-the-monsters.js

```diff
@@ -30,9 +30,11 @@ const scenes = {
 scenes.main.preload = function() {
     this.load.image("player", "sprites/players/mvp_player/idle_0.png");
```

Comment:
**issue**: The `workaroundSpawnGlitch` function is added to handle visibility issues. Verify that this does not introduce any new bugs or side effects.